app/authentication.go

```diff
@@ -0,0 +1,88 @@
+package app
+
+import (
+	"emreddit/db"
+	"emreddit/logger"
+	"errors"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+var KEY = "THIS IS A KEY"
```

Comment:
you shouldn't keep things like this key as an hardcoded. You can move these kind of values into `env variables`.

Checkout this package: https://github.com/go-nv/goenv

Reply:
same thing applies the things like database options, application port (which you gave fiber to listen)